openshift-kube-controller-manager/servicecacertpublisher/publisher.go

```diff
@@ -155,18 +158,52 @@ func (c *Publisher) processNextWorkItem() bool {
 	return true
 }
 
+var (
+	// default secure
+	// This annotation prompts the service ca operator to inject
+	// the service ca bundle into the configmap.
+	injectionAnnotation = map[string]string{
+		"service.beta.openshift.io/inject-cabundle": "true",
+	}
+	setAnnotationOnce = sync.Once{}
+)
+
+func getInjectionAnnotation() map[string]string {
+	setAnnotationOnce.Do(func() {
+		// this envvar can be used to get the kube-controller-manager to inject a vulnerable legacy service ca
+		// the kube-controller-manager carries no existing patches to launch, so we aren't going add new
+		// perma-flags.
+		// it would be nicer to find a way to pass this more obviously.  This is a deep side-effect.
+		// though ideally, we see this age out over time.
+		useVulnerable := os.Getenv("OPENSHIFT_USE_VULNERABLE_LEGACY_SERVICE_CA_CRT")
+		if len(useVulnerable) == 0 {
+			return
+		}
+		useVulnerableBool, err := strconv.ParseBool(useVulnerable)
```

Comment:
nit: s/useVulnerableBool/useVulnerable/ ? A name that indicates a question generally obviates the need for a type suffix indicating boolean.